backend/pyproject.toml

```
@@ -32,4 +36,30 @@ ensure_newline_before_comments = true
 line_length = 100
 
 [tool.mypy]
-ignore_missing_imports = true
\ No newline at end of file
+ignore_missing_imports = true
+
+[tool.poetry]
+name = "backend"
```

openfoodfacts-taxonomy-editor seems better to me :-) ? Or should it map the folder name (in which case this is ok).
```suggestion
name = "openfoodfacts-taxonomy-editor"
```